docs/css/store.scss

```diff
@@ -402,6 +414,113 @@ input, textarea, keygen, select, button {
   }
 }
 
+.Overlay {
+  background-color: rgba(43,51,61, 0.8);
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 2;
+  display: none;
+
+  &.isShown {
+    display: block;
+  }
+}
+
+.Modal {
+  background-color: $alabaster;
+  border-radius: 6px;
+  height: 480px;
+  width: 533px;
+  text-align: center;
+  padding-top: 53px;
+  margin: 5% auto 0;
```

Comment:
Please use `vh` for vertical percentages. Otherwise css calculates it based on width.